data/global_team/current/sara_acevedo.json

```diff
@@ -0,0 +1,8 @@
+  {
+    "name": "Sara Acevedo",
+    "role": ["CoC Enforcement Support"],
+    "start": "June 2024",
```

Comment:
is she not starting before the coming summer? Maybe it should say June 2023, since that's when she was offered the role?

Reply:
ohhhh you are right. I corrected it to 2023 😅